source/cards/deck.py

```diff
@@ -20,7 +20,7 @@ def build_initial_deck():
     return (reduce(lambda x, y: x + y,
                    [build_color_cards(color) for color in COLORS]) +
             [Card(CardType.CHANGE_COLOR, '', '')] * 2 +
```

Comment:
According to the rules, there should be 4 change_color cards

Reply:
It is later multiplied by 2, which in turn makes it 4

Reply:
Yeah I noticed that after I commented, got confused by the brackets

Reply:
Legit